<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResolutionInstance</class>
 <widget class="QDialog" name="ResolutionInstance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Résolution d'instances</string>
   </property>
  </widget>
  <widget class="QLabel" name="fichierInstanceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fichier d'instance :</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeResolutionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type de résolution :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeResolutionComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Exacte indéxée temps</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exacte indéxée jobs</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Heuristiques</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="choisirFichierLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="choisirFichierPushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Choisir un fichier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="validerPushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Résoudre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annulerPushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="fichierRadioButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fichier d'instance</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="dossierRadioButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dossier d'instances</string>
   </property>
  </widget>
  <widget class="QComboBox" name="triComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CCmax basé sur la somme des ressources de chaque job</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CCmax basé sur la ressource maximale de chaque job</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Somme des ressources de chaque job</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Valeur moyenne des ressources des sous-ensembles maximaux</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="typeTriLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type de tri :</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeAffectationLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type d'affectation :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="affectationComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Affectation machine par machine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Affectation priorisant la machine avec le moins de charge</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
